WordPress/Classes/ViewRelated/AccountSettingsViewController.swift

```diff
@@ -3,12 +3,14 @@ import UIKit
 import RxSwift
 import WordPressComAnalytics
 
-func AccountSettingsViewController(account account: WPAccount) -> ImmuTableViewController {
-    let service = AccountSettingsService(userID: account.userID.integerValue, api: account.restApi)
+func AccountSettingsViewController(account account: WPAccount?) -> ImmuTableViewController {
+    let service = account.map({ account in
+        return AccountSettingsService(userID: account.userID.integerValue, api: account.restApi)
+    })
```

Comment:
> Map is mostly used for collections

You might be used to see it only on collections, but it's a concept that works for any kind of container (you can think of an optional as a collection limited to zero or one items). Collections are just the most obvious use case.

> You can drop the parenthesis

For this I like the style proposed [here](http://ericasadun.com/2015/11/17/a-handful-of-swift-style-rules-swiftlang/)

> The Rule of Kevin: “When a trailing closure argument is functional, use parentheses. When it is procedural, use braces.”

```swift
myCollection.map({blah}).filter({blah}).etc
myCollection.forEach {} // or 
dispatch_after(when, queue) {}
```

It makes sense because `map`/`filter`/`reduce` are things that you often chain together, and this doesn't look as good: 

```swift
foo.map {
  return $0 * 2
}.filter {
  return $0 < 10
}
```